tests/unit_tests/test_wave_06.py

```diff
@@ -110,16 +125,29 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
```

Comment:
Prefer this assertion to explicitly check that `result` is `True` instead of checking that `result` is a truthy value. 

If `result` was a nonempty string/list/dictionary, then line 136 would 'pass'.

```suggestion
    assert result == True
```

Reply:
Updated the assertion to explicitly check that result is True instead of just a truthy value:
**assert result is True**